prow/tide/tide.go

```diff
@@ -246,11 +247,34 @@ func requirementDiff(pr *PullRequest, q *config.TideQuery) (string, int) {
 			desc = fmt.Sprintf(" Should not have %s labels.", strings.Join(trunced, ", "))
 		}
 	}
+	diff = len(missingLabels) + len(presentLabels)
+
+	if desc == "" {
```

Comment:
We still need to check for unsuccessful contexts in order to return a correct `diff` even if we already have a message to use for the status description.